templates/main/jams/profile.html

```diff
@@ -75,19 +86,47 @@ <h1 class="uk-header uk-article-title">
 
 
 <script type="application/javascript">
-    const date = flatpickr("#dob", {enableTime: false, altInput: true});
+    const date = flatpickr("#dob", {
+        enableTime: false, altInput: true, altInputClass: "date-picker",
+        onChange: function() {
+            let dob = moment(dob_input.value);
+
+            if (!dob.isBefore(earliest_dob)) {
+                UIkit.notification({
+                    "message": "You must be aged 13 or older to participate in our code jams.",
+                    "status": "danger",
+                    "pos": "top-center",
+                    "timeout": 5000,
+                });
+
+                dob_output.classList.add("uk-form-danger");
+                return submit_button.disabled = true;
```

Comment:
what

Reply:
I figured it was more concise than:

```js
submit_buton.disabled = true;
return;
```

Reply:
no no no no no no no no no. why do you even need to return at all?

Reply:
Oh, good point. Blame copypasta.